ddtrace/settings/_otel_remapper.py

```diff
@@ -0,0 +1,184 @@
+import os
+
+from ..internal.logger import get_logger
+
+
+log = get_logger(__name__)
+
+
+OTEL_UNIFIED_TAG_MAPPINGS = {
+    "deployment.environment": "DD_ENV",
+    "service.name": "DD_SERVICE",
+    "service.version": "DD_VERSION",
+}
+
+
+def _remap_otel_log_level(otel_value):
+    """Remaps the otel log level to ddtrace log level"""
+    if otel_value == "debug":
+        return "True"
+    elif otel_value == "info":
+        return "False"
+    else:
+        log.warning(
+            "ddtrace does not support otel log level '%s'. setting ddtrace to log level info.",
+            otel_value,
+        )
+        return "False"
+
+
+def _remap_otel_propagators(otel_value):
+    """Remaps the otel propagators to ddtrace propagators"""
+    accepted_styles = []
+    for style in otel_value.split(","):
+        style = style.strip().lower()
+        if style in ["b3", "b3multi", "b3single", "datadog", "tracecontext", "none"]:
+            if style == "b3single":
+                style = "b3"
+            if style not in accepted_styles:
+                accepted_styles.append(style)
+        else:
+            log.warning("Following style not supported by ddtrace: %s.", style)
+    return ",".join(accepted_styles)
+
+
+def _remap_traces_sampler(otel_value):
+    """Remaps the otel trace sampler to ddtrace trace sampler"""
+    if otel_value == "always_on" or otel_value == "parentbased_always_on":
```

Comment:
There's a subtle add-on we'll need to perform here: `always_on`, `always_off`, and `traceidratio` will also set a _brand new_ config flag called `DD_TRACE_SAMPLE_IGNORE_PARENT` to the value of `true`. There's two changes here that will need to happen (maybe we can do it in a separate PR)

1. The one OTEL env can set multiple DD envs
2. We need to implement and document the new DD config `DD_TRACE_SAMPLE_IGNORE_PARENT`

Let me know if I can help with these

Reply:
Will do this in a separate PR!